test/chaat/db/user_test.clj

```diff
@@ -0,0 +1,71 @@
+(ns chaat.db.user-test
+  (:require [chaat.fixture :as fixture]
+            [clojure.test :refer :all]
+            [chaat.db.user :as db.user]
+            [chaat.model.user :as model.user]
+            [crypto.password.bcrypt :as bcrypt]
+            [java-time.api :as jt]))
+
+(use-fixtures :each fixture/test-fixture)
+
+(defn- time-stub
+  "Return a fixed timestamp"
+  []
+  (jt/instant "2023-06-13T10:07:03.172Z"))
+
+(defn- encrypt-stub
+  "Return a fixed password hash"
+  [_ _]
+  "$2a$11$DoWjFwnL5glpyGqBRgdA3uqoy1glTFVoXP.wesem27g2SL3XFXOHW")
+
+;; no need to test this?
+(deftest new-user?-test
+  (testing ""))
+
+;; no need to test this?
+(deftest user-exists?-test
+  (testing ""))
```

Comment:
These need to be addressed. For now, I think it's okay to not test these, but on the other hand, it should be quite easy to test, so let's just do it.

Reply:
I've decided to make these functions private. Also, testing the other functions which use these 2 simple functions will cover this.